src/convertToObject.js

```diff
@@ -1,18 +1,20 @@
-'use strict';
+"use strict";
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(";");
+  const stylesCSS = {};
+
+  styles
+    .filter((style) => (style !== "" ? style.trim() : false))
```

Comment:
```suggestion
    .filter((style) => style.trim())
```
You don't ternary operator in this case, and compare with an empty string
https://developer.mozilla.org/en-US/docs/Glossary/Truthy